README.md

```diff
@@ -59,7 +59,8 @@ Role Variables
       defined with the following dict:
         - `pool`: Name or UUID of the storage pool from which the volume should be
           allocated.
-        - `name`: Name to associate with the volume being created.
+        - `name`: Name to associate with the volume being created; include extension if you would like images created with one.
+        - `file_path`: Where the storage file should be placed under; defaults to `libvirt_volume_default_images_path`
```

Comment:
Can you make it clear this is for type=file only?

Reply:
I've tried to improve this with 6356e58